client/Components/Chat/Convo/Dialog.tsx

```diff
@@ -32,7 +32,10 @@ function DialogComponent( {
   return (
     <InnerContainer className="flex-col-reverse" onTop={ handleScrollToTop }>
       { ! conversation.length ? (
-        <ChatWelcome user={ page.user } />
+        <>
+          { alertMessage ? alertMessage : null }
+          <ChatWelcome user={ page.user } />
+        </>
       ) : (
         <>
           { alertMessage ? alertMessage : null }
```

Comment:
Let's move  `{ alertMessage ? alertMessage : null }` right under InnerContainer to avoid repeating it

Reply:
Done